### ISSUE TITLE
How to build, distribute and run clients

### ISSUE THREAD

**Issue by SamBouwer** [SamBouwer/any-docker#2]
Try to script and add instructions how to build your own Anytype client from source to connect to your selfhosted node

- [x] Build anytype-heart middleware libraries for Desktop
- [ ] Build anytype-heart middleware libraries for Android
- [ ] Build anytype-heart middleware libraries for iOS
- [x] Build anytype-ts Desktop client for Windows
- [x] Build anytype-ts Desktop client for Linux
- [ ] Build anytype-ts Desktop client for MacOS
- [ ] Update docs

**Comment by ahmedwael216** [comment]
Just a note, on a future release the clients built and distributed by AnyType will support changing the network to a local one, so this is not *really* required.

See [this](https://github.com/anyproto/contributors/discussions/17#discussioncomment-6488875).

**Comment by SamBouwer** [comment]
You are right, but I'm not sure what will be there first (docker image vs custom storage location). As I want to become familiar with docker image building instead of only leeching of the work of others, I want to give it a try with the current Anytype version.

Thanks for thinking along!
